src/rdkafka_topic.h

```diff
@@ -104,6 +104,10 @@ struct rd_kafka_topic_s {
 	rd_kafka_toppar_t **rkt_p;          /**< Partition array */
 	int32_t            rkt_partition_cnt;
 
+        int32_t            rkt_sticky_par;    /* Sticky partition */
```

Comment:
To keep with current naming strategies, use rkt_sticky_partition.

Also have the comments be doxygen formatted:
```c
int32_t        rkt_sticky_partition;   /**< Current sticky partition.
                                        *     @locks rkt_lock */
rd_interval_t .......                  /**< .. .. */
```